addons/mail/static/src/core/common/chat_hub.xml

```diff
@@ -2,7 +2,7 @@
 <templates xml:space="preserve">
 
 <t t-name="mail.ChatHub">
-    <div class="o-mail-ChatHub" t-if="isShown or ui.isSmall">
```

Comment:
Wasn't it the only usage of `isShown`? If it is removed there's no point to keep it.